ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1038 @@
+|===
+|Project |Description
+
+|`client`
+|An application that you deploy to the first server, to `server1`.
+It provides REST endpoints that provides EJB remote calls to the `server application`
+residing on the other servers.
+Besides the remote EJB call the REST endpoint invocation processes data insertion to a database.
+As the REST invocation starts a transaction then the transaction enlists two participants
+(the database and the remote server called with EJB remote invocation).
+The transaction manager then uses the two-phase commit for committing the transaction.
+The quickstart examines failures and shows the transactional behaviour
+in such cases.
+
+|`server`
+|An application receives the remote EJB calls from the `client` application deployed at `server1`.
+This application is deployed to `server2` and `server3`.
+The EJB component receives the EJB remote call and depending on the scenario
+it resumes on the transaction propagated in the context of the remote EJB call.
+The call inserts data to the database.
+The applications also enlists a special XAResource to the transaction to demonstrate the quickstart.
+|===
+
+=== What to expect?
+
+The description is separated into two parts. The first one explains how to run the quickstart in a bare metal environment.
+The second one talks about deployment of the quickstart to Kubernetes and to OpenShift.
+
+The sequence of events to expected for bare metal installation:
+
+. Create three directories `server1`, `server2` and `server3` from installation directory of {productName}
+. Configure credentials on `server2` and `server2`
+. Configure remote outbound connection from `server1` to both servers `server2` and `server3`
+. Configure datasources, first on `server1` and then on both servers `server2` and `server3`
+. Start all 3 servers
+. Deploy the quickstart application to all servers.
+.. The deployed application at `server1` behaves as a client to both other servers.
+The `server1` offers several HTTP endpoints <<rest-endpoints>>. When the HTTP endpoint is invoked
+it starts an EJB call going to one of the servers `server2` or `server3`.
+. Call one of the HTTP endpoints at `server1`. Based on the scenario EJB remote calls
+hit the remote server and specific behaviour can be observed (see<<rest-endpoints>>).
+.. During the EJB remote calls data is inserted into database at the each involved server.
+
+The sequence of events expected for Kubernetes deployment:
+
+. Start PostgreSQL database in the Kubernetes
+. Build the docker image from this quickstart with `s2i`
+. Push the docker image to Kubernetes registry
+. Deploy the {productName} Operator to Kubernetes
+. Deploy the `CustomerResource` definition of this quickstart to Kubernetes
+. Run the HTTP calls against `server1` that invoke the EJB remote calls to other two servers.
+Then examine the behaviour
+
+The sequence of events expected for OpenShift deployment:
+
+. Start PostgreSQL database in the OpenShift
```

Comment:
Article issues.

All steps int eh procedure are missing periods at the end of the sentences.